frontend/graphql/GetPages.graphql

```diff
@@ -1,43 +1,55 @@
 query GetPages($where: JSON, $locale: String) {
   pages(where: $where, locale: $locale) {
     id
-    locale
-    created_at
-    updated_at
-    pageName
     path
-    blocks {
-      __typename
-      ... on ComponentBlocksCard {
+    pageName
+    locale
+    __typename
+    sections {
+      ... on ComponentSectionSingleFeatureSection {
+        __typename
         id
+        sectionTitle
         title
-        description
-        image {
-          url
-        }
-        projectLink {
-          path {
-            path
+        subtitle
+        singleFeature: sections {
+          __typename
+          id
+          description
+          title
+          serviceLink
+          image {
+            id
+            name
+            alternativeText
+            width
+            height
+            url
           }
         }
       }
-      ... on ComponentBlocksSingleFeature {
+      ... on ComponentSectionHeroSection {
+        __typename
         id
         title
-        description
-        image {
-          url
-        }
-        serviceLink {
-          path {
-            path
-          }
-        }
+        subtitle
       }
-      ... on ComponentBlocksHero {
+      ... on ComponentSectionCardSection {
+        __typename
         id
+        sectionTitle
         title
         subtitle
+        card: sections {
+          id
+
+          title
+          description
+          image {
+            url
+          }
+          projectLink
```

Comment:
Why `projectLink`? If we make the data component generic, we shouldn't encode specific use cases as names for the field... I'd put just `url`